### PR TITLE
[T2 profile]: Skipping checks for psus part of absent psu list (Cisco-8000)

### PR DESCRIPTION
**Pull request by vperumal**
### Description of PR
Even though the script provides an option for skip_psu_list. It is used only for presence check. The test currently fails on distributed chassis, wherein it tries to check values for absent psus.

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
Skip psus that are absent in the chassis

#### How did you do it?
Check if the psus is part of skip_psu list. If yes, skip the test for that psu

#### How did you verify/test it?
Verified on Cisco-8000 platform

### CODE REVIEW

**Comment by vperumal** [comment]
FYI @abdosi

**Comment by vperumal** [comment]
Hi @gechiang , Kindly review the PR

File: tests/platform_tests/api/test_psu.py [hunk 1]
```diff
@@ -98,6 +98,9 @@ def test_get_name(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost, p
             if self.skip_absent_psu(i,platform_api_conn):
                 continue
             name = psu.get_name(platform_api_conn, i)
+            if name in self.psu_skip_list:
```
Comment (gechiang) [inline tests/platform_tests/api/test_psu.py:101]: isn't the purpose of calling "if self.skip_absent_psu(i,platform_api_conn):" in line 98 achieve the same as what you have added to check the name to skip over those on the skip list?  If so, why do you still need this check?
Can you check again why the previous check is not accomplishing the same as your newly added check?

Same comment for the rest of all your changes.  they all have the call to "if self.skip_absent_psu(i,platform_api_conn):" already before your change.

**Comment by vperumal** [comment]
Closing the PR. Since the functionality is present in the current code

**Pull request closed without merging**